homeassistant/components/lcn/__init__.py

```diff
@@ -56,6 +60,28 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if DOMAIN not in config:
         return True
 
+    _LOGGER.warning(
+        "Configuration of LCN integration in YAML is deprecated and "
+        "will be removed in a future release; Your existing configuration "
+        "has been imported into the UI automatically "
+        "and can be safely removed from your configuration.yaml file"
+    )
+    async_create_issue(
+        hass,
+        HOMEASSISTANT_DOMAIN,
+        f"deprecated_yaml_{DOMAIN}",
+        breaks_in_ha_version="2024.7.0",
```

Comment:
Let's bump this, maybe to 2024.12.0?
```suggestion
        breaks_in_ha_version="2024.12.0",
```

Reply:
Changed